services/121-service/package.json

```
@@ -29,6 +29,11 @@
     "test:integration:debug": "node --inspect-brk=0.0.0.0:9231 --require=tsconfig-paths/register --expose-gc node_modules/.bin/jest --config=jest.integration.config.js --runInBand --detectOpenHandles --logHeapUsage",
     "test:integration:watch": "npm run test:integration:all -- --watchAll",
     "test:integration:coverage": "npm run test:integration:all -- --coverage --coverageDirectory=coverage/integration",
+    "coverage:fix-paths": "find coverage -type f -name \"coverage-final.json\" -exec sed -i '' \"s|/home/node/app|$(pwd)|g\" {} \\;",
+    "coverage:merge": "istanbul-merge --out coverage/combined/coverage.json coverage/integration/coverage-final.json coverage/unit/coverage-final.json",
+    "coverage:report": "nyc report -t coverage/combined --report-dir coverage/combined --reporter=lcov --reporter=text",
+    "coverage": "npm run coverage:fix-paths && npm run coverage:merge && npm run coverage:report",
+    "open-coverage-report": "open coverage/combined/lcov-report/index.html",
```

```suggestion
    "coverage:open-report": "open coverage/combined/lcov-report/index.html",
```

I like namespacing things. ;)